util/progress/printer.go

```diff
@@ -41,7 +42,7 @@ func (p *Printer) Warnings() []client.VertexWarning {
 	return p.warnings
 }
 
-func NewPrinter(ctx context.Context, out console.File, mode string) *Printer {
+func NewPrinter(ctx context.Context, w io.Writer, mode string) (*Printer, error) {
```

Comment:
Would passing console as a second parameter be suitable here? I don't want the tty mode to silently get disabled when some change starts to pass wrong parameter.

Reply:
Yeah that works for me.